src/jade/splash/project_vm.jade

```diff
@@ -1,4 +1,21 @@
 div.editor-container
+
+    div.editor-header
+        button.btn.btn-primary.toolbar-btn(type="button", data-toggle="tooltip", data-placement="bottom", title="Save", rv-command-save) 
```

Comment:
You should use a mixin (or a loop maybe?) to dry this up.  There is a lot of duplication.